src/mumble/ConnectDialog.cpp

```diff
@@ -1627,17 +1622,15 @@ void ConnectDialog::startDns(ServerItem *si) {
 		foreach (const ServerAddress &addr, si->qlAddresses) { qhPings[addr].insert(si); }
 		return;
 	}
-
-#ifdef USE_BONJOUR
-	if (bAllowBonjour && si->qsHostname.isEmpty() && !si->brRecord.serviceName.isEmpty()) {
-		if (!qlBonjourActive.contains(si->brRecord)) {
-			g.bc->bsrResolver->resolveBonjourRecord(si->brRecord);
-			qlBonjourActive.append(si->brRecord);
+#ifdef USE_ZEROCONF
+	if (bAllowBonjour && si->qsHostname.isEmpty() && !si->zeroconfRecord.serviceName.isEmpty()) {
```

Comment:
`bAllowBonjour` seems to be a misleading name with these changes in place :thinking:

Reply:
I agree. We can safely rename it because it's not a configuration option, it's simply set to `false` whenever a proxy is used.